### ISSUE TITLE
pdfbox@2.0.22: Download via aria2 failed

### ISSUE THREAD

**Issue by digrouz** [ScoopInstaller/Main#1965]
```
Download failed! (Error 3) Resource was not foundInstalling 'pdfbox' (2.0.22) [64bit] [main]
WARN  By installing you accept following license: Apache-2.0 (https://spdx.org/licenses/Apache-2.0.html)
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: f3f054|ERR |       0B/s|D:/users/digrouz/scoop/cache/pdfbox#2.0.22#https_www.apache.org_dist_pdfbox_2.0.22_preflight-app-2.0.22.jar_preflight.jar
Download: a0b3a7|ERR |       0B/s|D:/users/digrouz/scoop/cache/pdfbox#2.0.22#https_www.apache.org_dist_pdfbox_2.0.22_pdfbox-app-2.0.22.jar_pdfbox.jar
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://www.apache.org/dist/pdfbox/2.0.22/pdfbox-app-2.0.22.jar#/pdfbox.jar
    referer=https://www.apache.org/dist/pdfbox/2.0.22/pdfbox-app-2.0.22.jar#/
    dir=D:\users\digrouz\scoop\cache
    out=pdfbox#2.0.22#https_www.apache.org_dist_pdfbox_2.0.22_pdfbox-app-2.0.22.jar_pdfbox.jar
https://www.apache.org/dist/pdfbox/2.0.22/preflight-app-2.0.22.jar#/preflight.jar
    referer=https://www.apache.org/dist/pdfbox/2.0.22/preflight-app-2.0.22.jar#/
    dir=D:\users\digrouz\scoop\cache
    out=pdfbox#2.0.22#https_www.apache.org_dist_pdfbox_2.0.22_preflight-app-2.0.22.jar_preflight.jar

ERROR & 'D:\users\digrouz\scoop\apps\aria2\current\aria2c.exe' --input-file='D:\users\digrouz\scoop\cache\pdfbox.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=22152 --continue --summary-interval 0
ERROR Download failed! (Error 3) Resource was not found

```

**Comment by digrouz** [comment]
see PR #1966

**Comment by akirayamamoto** [comment]
This is fixed on #1966
Thank you for contributing.
